File: Random Code Challenges/transpose.js
function transpose(array){
  // create a new array to push our answer into
  let answer = [];
  // find the max length of either string
  let maxLength = Math.max(array[0].length, array[1].length);
  // iterate through our strings
  for(let i = 0; i < maxLength; i++)
    // push the character into our array. IF there is no character to be pushed we push in a space
    answer.push((array[0][i] || " ") + " " + (array[1][i] || " "));
  // combine our characters together with a new line inbetween ever character
  answer.join('/n');
  
  // return answer
  return answer;
}

File: Random Code Challenges/Transpose.java
    public static void transpose(String[] words){
        // Find the maximum length of either word1 or word2
        int max = Math.max(words[0].length(), words[1].length());
        
        // If word1 is shorter than word2, continue to add spaces to word1 until their length is equal
        while(words[0].length() < words[1].length())
            words[0] += " ";
        
        // If word2 is shorter than word1, continue to add spaces to word2 until their length is equal
        while(words[1].length() < words[0].length())
            words[1] += " ";
        
        // Print out each character of each word and a new line at the end of every iteration
        for(int i = 0; i < max; i++)
            System.out.print(words[0].charAt(i) + " " + words[1].charAt(i) + "\n");
    }
